sequelizeDatabase/sequelFunctions.js

```diff
@@ -1,350 +1,481 @@
+      {
+        model: models.Companies
+      }
+    ]
+  }).then(wholeHistory => {
+    return wholeHistory;
+  });
+};
 
-getIndividualCurrentEmployement = (IndividualID) => {
-    return getIndividualEmployeeHistory(IndividualID).then((IndividualHistory) => {
-        return IndividualHistory[0];
-    })
-}
+getIndividualCurrentEmployement = IndividualID => {
+  return getIndividualEmployeeHistory(IndividualID).then(IndividualHistory => {
+    return IndividualHistory[0];
+  });
+};
 
 //---------------Modify Existing Data Functions----------//
 modifyIndividual = (IndividualID, newName, newPosition, newUrl, newComments) => {
-    return models.Individuals.findOne({
-        where: {
-            IndividualID: IndividualID
-        }
-    }).then(individual => {
-        individual.update({
-            IndividualName: newName,
-            OriginalPostion: newPosition,//TODO: Update this.
-            LinkedInUrl: newUrl,
-            Comments: newComments
-        })
-        return individual;
-    }).catch(err => console.error(err));
-}
+  return models.Individuals.findOne({
+    where: {
+      IndividualID: IndividualID
+    }
+  })
+    .then(individual => {
+      individual.update({
+        IndividualName: newName,
+        OriginalPostion: newPosition, //TODO: Update this.
+        LinkedInUrl: newUrl,
+        Comments: newComments
+      });
+      return individual;
+    })
+    .catch(err => console.error(err));
+};
 
-deleteIndividual = (IndividualID) => {
-    // I can't get the delete for an individual to in turn
-    //   delete rows from EmployeeHistory, so I'm doing 
-    //   this the long way until I can ask about Sequelize
-    //   Cascade. --Sean
-    //
-    // Modfying this to avoid a race condition -- Aaron
-    return new Promise((resolve,reject)=>{
-        models.EmployeeHistory.destroy({
-            where: {
-                IndividualID: IndividualID
-            }
-        }).then((deletedHistory) => {
-            models.Individuals.destroy({
-                where: {
-                    IndividualID: IndividualID
-                }
-            }).then((deletedIndividual) =>{
-                console.log("deletion resolved")
-                resolve(deletedIndividual);
-            })
-        }).catch(err => {console.error(err)
-                        reject(err)
-        })
+deleteIndividual = IndividualID => {
+  // I can't get the delete for an individual to in turn
+  //   delete rows from EmployeeHistory, so I'm doing
+  //   this the long way until I can ask about Sequelize
+  //   Cascade. --Sean
+  //
+  // Modfying this to avoid a race condition -- Aaron
+  return new Promise((resolve, reject) => {
+    models.EmployeeHistory.destroy({
+      where: {
+        IndividualID: IndividualID
+      }
     })
-}
+      .then(deletedHistory => {
+        models.Individuals.destroy({
+          where: {
+            IndividualID: IndividualID
+          }
+        }).then(deletedIndividual => {
```

Comment:
**ISSUE:** Need to update SQL Tables to have Cascade so that these delete functions can be simplified.